### DISCUSSION TITLE
StackOverflow

### DISCUSSION THREAD

**Discussion by martenw** [abpframework/abp#6218] [General]
Hi

What are your thoughts on the [abp-tag on StackOverflow](https://stackoverflow.com/questions/tagged/abp)? Many questions there seem to be left unanswered, is that because they are very complex or is that forum not monitored? Is there any other better place to ask general questions on how to use the framework for us not using commercial? 

/Mårten

**Comment by 274188A** [comment]
hard to  find the time to be honest...

**Comment by hikalkan** [comment]
You can ask to stackoverflow or here. However, we can't answer all the questions, to be honest. It gets a lot of time. We are trying to do our best to answer all the questions everywhere. Some questions are hard to understand and re-produce and especially some of the questions are like "consultancy" style questions, some of them are general application development requirements, but it is not easy to give free consultancy to everyone 😄
ABP Commercial [support web site](https://support.abp.io/QA/Questions) has high priority since they are paid customers.
